docs/data/base/components/autocomplete/autocomplete.md

```diff
@@ -14,6 +14,8 @@ waiAria: https://www.w3.org/WAI/ARIA/apg/patterns/combobox/
 
 {{"component": "modules/components/ComponentPageTabs.js"}}
 
+- 📦 [4.5 kB gzipped](/size-snapshot/).
```

Comment:
The size-snapshot doesn't look very user-frienly. Plus, it looks outdated, as the first line says "@material-ui/core". Linking to Bundlephobia instead will result in a better UX.

Reply:
Then?

```suggestion
- 📦 [4.5 kB gzipped](https://bundlephobia.com/package/@mui/base).
```

Reply:
Yup, looks better to me!